EddystoneBeacon.cpp

```diff
@@ -76,6 +76,33 @@ void EddystoneBeacon::begin(char power, const char* uri) {
 
   this->_bleCharacteristic.broadcast();
 }
+/***Following is the code for broadcasting TLM ****/
+/***Implementation in the examples folder***/
+uint8_t buffer[14]= {
+  0x20,
+  0x00,
+  0x00,
+  0x00,
+  0x00,
+  0x80,
+  0x00,
+  0x00,
+  0x00,
+  0x00,
+  0x00,
+  0x00,
+  0x00,
+  0x00
+}; //global
+unsigned char len = 14; //global
+void EddystoneBeacon::tlm_begin(){
```

Comment:
Having declared and defined `uint8_t buffer[14]` in` Eddystone.h` I changed `tlm_begin()` to the following:
void EddystoneBeacon::begin(float temp, float bLevel){
  
  this->_bleCharacteristic.setValue(buffer, len);

  BLEPeripheral::begin();

  this->_bleCharacteristic.broadcast();

}

I have added `setTLMData(float temp,float bLevel)` to set the TLM values.